run.cmd

```diff
@@ -8,6 +8,5 @@
 @ECHO CBScript 1.20
 @title %~nx1
 @cd "%~dp0"
-@if not defined PYTHON27 set PYTHON27=c:\python27\python
-py -2.7 compile.py %1
```

Comment:
maybe chance this to -3 instead of removing it, ensuring that py won't try running it using python2 if that is set as the default.

Reply:
I agree with this.